vector-th-unbox.cabal

```diff
@@ -17,10 +17,17 @@ category:       Data
 build-type:     Simple
 cabal-version:  >= 1.10
 tested-with:
-  GHC == 8.0.2,
-  GHC == 8.2.2,
-  GHC == 8.4.4,
-  GHC == 8.6.5, GHC == 8.8.3, GHC == 8.10.1, GHC == 9.0.1, GHC ==9.2.1
+  GHC == 8.0,
+  GHC == 8.2,
+  GHC == 8.4,
+  GHC == 8.6,
+  GHC == 8.8,
+  GHC == 8.10,
+  GHC == 9.0,
+  GHC == 9.2,
+  GHC == 9.4,
+  GHC == 9.6,
+  GHC == 9.8
```

Comment:
Is there a pretext for this form of constraint (`GHC == x.y`)?  E.g. I am not sure `haskell-ci` would understand it.  
I have seen `GHC == 8.0.*` before, meaning `GHC == { 8.0.1, 8.0.2 }`.
Alas, `cabal check` does not check these constraints, so there is wild growth...

Reply:
Ah, good point, I just wanted to make it consistent with the specification in the GHA `haskell.yml`. Happy to add back the third component.

According to https://hackage-search.serokell.io/?q=GHC+%3D%3D+.%5C...%3F%28%2C%7C%24%29, there are already lots of packages that use this style:

> Search completed (238 matches across 90 packages)

Reply:
Yeah, suppose this can be considered a kind of "free-form" field for most cases.